packages/fluentui/e2e/package.json

```diff
@@ -9,7 +9,7 @@
     "@fluentui/react-northstar": "^0.61.0",
     "@fluentui/react-northstar-prototypes": "^0.61.0",
     "@types/react-router-dom": "^5.1.5",
-    "cypress": "6.6.0",
+    "cypress": "9.5.2",
```

Comment:
single version of cypress in the repo, there aren't any huge breaking changes I'm aware of.